lms/templates/admin/organization.html.jinja2

```diff
@@ -1,5 +1,24 @@
 {% import "macros.html.jinja2" as macros %}
 
+{% macro org_hierarchy(node, focus_org) %}
+    <li>
+        {% if node.id == focus_org.id %}
+            <b>(You are here)</b> {{ node.name }} - <code>{{ node.public_id }}</code>
+        {% else %}
+            <a href="{{ request.route_url('admin.organization', id_=node.id) }}">{{ node.name }}</a>
```

Comment:
We display the `node.name` for "you are here" case but not the others. It might be more valuable the other way around.